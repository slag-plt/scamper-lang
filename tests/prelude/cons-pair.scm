(cons #t #f)
(cons 1 2)
(cons "hi" "bye")
(cons "a" "b")
(cons 0.003 100)
(pair 1 2)
(pair #t #f)
(pair "hi" "bye")
(pair "a" "b")
(pair 0.003 100)