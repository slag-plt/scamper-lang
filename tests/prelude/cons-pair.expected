(cons #t #f)
(cons 1 2)
(cons "hi" "bye")
(cons "a" "b")
(cons 0.003 100)
(cons 1 2)
(cons #t #f)
(cons "hi" "bye")
(cons "a" "b")
(cons 0.003 100)
